### DISCUSSION TITLE
Option to show current view name in place of Lovelace title

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11228] [Other feature requests]
## The request

At the top of your screen is the "Title of your Lovelace UI" 

![default](https://i.imgur.com/f17H1AG.png)

I think a cool **option** might be to display the active view name in place of that. 

![view](https://i.imgur.com/6FJsI9F.png)

When you click the edit pencil icon next to the title, underneath the text field to rename it, would be a toggle. When this is toggled on it, it would display the text field in a disabled / grayed out state to indicate it's not being used.

![current-view](https://i.imgur.com/qobeIbC.jpg)

**Comment by zsarnett** [comment]
Would it be better to append the view name here?

`Home - Lights View`

**Comment by Mariusthvdb** [comment]
Yes, the append sounds great because that would allow for easier distinction between different instances.

Have the instance name appended with the view title. At least as option please, for those that have more than 1 instance (using 3 here and heavily relying on that naming convention, which now is possible using CustomHeader and FavIcon)

![image](https://user-images.githubusercontent.com/33354141/94986269-b0214580-055d-11eb-9c57-623b3bbdd77a.jpeg)

![image](https://user-images.githubusercontent.com/33354141/94986272-b4e5f980-055d-11eb-85e7-97e919f3c974.jpeg)

**Comment by danimart1991** [comment]
I would like to take this issue.

**Comment by danimart1991** [comment]
According to the comments in the Issue.

What do you think about a first approach appending the view name to the title automatically if more than 1 view configured?
